docs/processes/release/tasks/versions.md

```diff
@@ -92,6 +92,126 @@ Edit icuver.txt directly.
 
 ### ICU4J
 
+#### Since ICU 74
+
+Changing the version for Java starting with ICU 74 requires a few steps:
+
+1. [icu4j/main/core/src/main/java/com/ibm/icu/util/VersionInfo.java](https://github.com/unicode-org/icu/blob/main/icu4j/main/core/src/main/java/com/ibm/icu/util/VersionInfo.java)
```

Comment:
Nitpick for step 1: it is unclear what one should do here.

Reply:
We are just changing the `ICU_VERSION` right?

Reply:
Done. Updated docs here and also below (where I copied it from).